.github/workflows/clean_up.yml

```diff
@@ -42,7 +42,7 @@ jobs:
               /repos/nvidia/cuda-quantum/git/refs/heads/$branch 
           done
         env:
-          GH_TOKEN: ${{ github.token }}
+          GH_TOKEN: ${{ secrets.REPO_BOT_ACCESS_TOKEN }}
```

Comment:
Is this new secret pointing back to the public repo? If so, what (if anything) would delete the bot/* branches in the private repo?

Reply:
this is a secret on the repository that the workflow runs on. I can make this more fork-friendly by falling back to the gh token if it is not defined.